plugin/storage/es/esRollover.py

```diff
@@ -227,6 +251,17 @@ def create_client(username, password, tls, ca, cert, key, skipHostVerify):
         return elasticsearch.Elasticsearch(sys.argv[2:], ssl_context=context)
 
 
+def check_if_ilm_policy_exists(ilm_policy):
+    """"
+    Checks whether ilm is created in Elasticsearch
+    """
+    s = get_request_session(os.getenv("ES_USERNAME"), os.getenv("ES_PASSWORD"), str2bool(os.getenv("ES_TLS", 'false')),
+                            os.getenv("ES_TLS_CA"), os.getenv("ES_TLS_CERT"), os.getenv("ES_TLS_KEY"),
+                            os.getenv("ES_TLS_SKIP_HOST_VERIFY", 'false'))
+    r = s.get(sys.argv[2] + '/_ilm/policy/' + ilm_policy)
+    if r.status_code != 200:
+        sys.exit("ILM policy '{}' doesn't exist in Elasticsearch. Please create it and rerun init".format(ilm_policy))
```

Comment:
If this fails on a user, will they know how to rerun init? If not, we should provide these instructions, or better yet, redirect them to https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/es/README.md and add some clear instructions there.

Reply:
Has this been addressed? If not, we could leave it for another PR (please create another issue for it). Given it's a new feature, it's important that its usage is well documented, along with troubleshooting steps.